mdal/frmts/mdal_ascii_dat.cpp

```diff
@@ -444,6 +444,77 @@ void MDAL::DriverAsciiDat::readFaceTimestep(
   group->datasets.push_back( dataset );
 }
 
+bool MDAL::DriverAsciiDat::persist( MDAL::DatasetGroup *group )
+{
+    std::ofstream out( group->uri(), std::ofstream::out );
```

Comment:
In the reader part of the code it looks for "_els_ in the file name ( current line 148). Currently when I am writing a file, I simply add _els_ at the end of the file name before the .dat file extension. Which is not really ideal since it gives files looking like this xxxyyy_els_.dat. I am assuming _els_ position is not actually defined and it can be anywhere.

If we can pick, we should probably agree on something better, either adding _els_ into another part of the file name or adding just _els at the end? (xxxyy_els.dat - looks much better),

Reply:
it is ok at the end